Show current shiny Pokemon in the breeding list, Update hatch button alignment

RedSparr0w opened the pull request:
This PR will show the shiny sprite in the breeding list, and also adds a sparkle emoji next to the name.
_note that the sparkle icon will look slightly different between different browsers_
![image](https://user-images.githubusercontent.com/7288322/58293527-db4c3e80-7e19-11e9-920c-6143fc970156.png)
![image](https://user-images.githubusercontent.com/7288322/58293605-531a6900-7e1a-11e9-9d11-73a726b27cc7.png)
![image](https://user-images.githubusercontent.com/7288322/58296284-38022600-7e27-11e9-96d9-9608330ebca6.png)

Fixed the alignment of the Hatch button and progress bars:
Added progress text to the progress bar
![image](https://user-images.githubusercontent.com/7288322/58292593-092f8400-7e16-11e9-9fbd-3e1d32c075cd.png)

Ishadijcks commented:
Looks good!

Any chance you can fix the hatching progress to stay centered regardless of progress? Currently it's
centered in the colored progress bar.

RedSparr0w commented:
Updated to always be centered on the entire progress bar instead of just in the colored part.
I've also removed the tooltip, as there isn't much point in showing that now that the progress % is always visible.

Ishadijcks commented on src/scripts/breeding/BreedingHelper.ts:
```diff
@@ -13,8 +13,8 @@ class BreedingHelper {
 
     public static progressEggs(amount: number) {
-        if (OakItemRunner.isActive("Blaze Casette")) {
-            amount *= (1 + OakItemRunner.calculateBonus("Blaze Casette") / 100)
```
#297
